t5/data/mixtures.py

```diff
@@ -26,23 +26,47 @@
 import tensorflow_datasets as tfds
 
 
+_SUPER_GLUE_WEIGHT_MAPPING = {
```

Comment:
Suggest adding comment stating that these mixture weights are based on the number of examples in each dataset.